packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap

```diff
@@ -544,7 +544,7 @@ exports[`StatsTestCases should print correct stats for filename 1`] = `
   "errors": [],
   "errorsCount": 0,
   "filteredModules": undefined,
-  "hash": "d11c74c648d060ebcdfd",
+  "hash": "1019e65f759482cee6e5",
```

Comment:
@xc2 can you help whey the hash changed here? did the minify logic change for this case?

Reply:
still checking, might respond later tonight

- [x] what cases? looks like all simple cases with dynamic import, thus these hash changes might be `splitChunks` related or dynamic import related.
- [x] does the building artifacts (case `dist`) change? no
- [x] does the module order change? maybe no per the stats output.
- [ ] actual reason? checking, guess it being the plugins order change

Reply:
@xc2 do you find the reason?